client-runtime/src/main/java/com/microsoft/rest/v2/RestResponse.java

```diff
@@ -39,63 +34,30 @@ public RestResponse(int statusCode, THeaders headers, Map<String, String> rawHea
     }
 
     /**
-     * The status code of the HTTP response.
      * @return The status code of the HTTP response.
```

Comment:
Javadoc with repetition like this should be trimmed down. checkstyle requires the `@return` but not the description. Will figure out what the best practice is here.

Reply:
Anu mentioned this before. Many editors use the comment's description (the part you've removed) as the description in the Autocompletion/Intellisense popup. I believe the only place the `@return` piece is used is in Javadocs. I think it's a mistake to remove the description.

Reply:
OK, I think bringing it back for a better experience would be the right thing. I wonder which editors that's true for. In IntelliJ:
<img width="701" alt="screen shot 2018-03-05 at 10 15 42 am" src="https://user-images.githubusercontent.com/5833655/36991976-4775df50-205e-11e8-957c-fabb1ac0ed52.png">